src/inputs-controller.ts

```diff
@@ -312,7 +314,7 @@ export class InputsController {
       globalScene.gameData?.saveMappingConfigs(gamepadID, this.configs[gamepadID]);
     }
     this.lastSource = "gamepad";
-    const handler = globalScene.ui?.handlers[UiMode.SETTINGS_GAMEPAD] as SettingsGamepadUiHandler;
+    const handler = globalScene.ui?.handlers[UiMode.SETTINGS_GAMEPAD] as GamepadSettingsUiHandler;
```

Comment:
Q: We should possibly update the `Ui.getHandler<T>()` method to also allow a parameter. Thus we could call
```suggestion
    const handler = globalScene.ui?.getHandler<GamepadSettingsUiHandler>(UiMode.SETTINGS_GAMEPAD);
```
then